### PR TITLE
VDM-73 Added API related to rejoin.

### PR DESCRIPTION
**Pull request by deerwalk**

### CODE REVIEW

File: lib/python/vdm/server/HTTPListener.py [hunk 1]
```diff
@@ -1806,6 +1841,9 @@ def main(runner, amodule, config_dir, data_dir, server):
     APP.add_url_rule('/api/1.0/databases/<int:database_id>/servers/recover',
                      view_func=RECOVER_DATABASE_SERVER_VIEW, methods=['PUT'])
 
+    APP.add_url_rule('/api/1.0/databases/<int:database_id>/servers/rejoin',
```
Comment (manjujames) [inline lib/python/vdm/server/HTTPListener.py]: This should have server id in the URL similar to START_DATABASE_SERVER_VIEW url. serverId should NOT be a request parameter.

File: lib/python/vdm/server/HTTPListener.py [hunk 2]
```diff
@@ -1375,8 +1375,12 @@ def put(database_id, server_id):
         """
 
         try:
+            if 'blocking' in request.args:
```
Comment (manjujames) [inline lib/python/vdm/server/HTTPListener.py]: Why blocking arg in start server api?

**Comment by manjujames** [comment]
I don't think there is a need for new rejoin endpoint. The point of this ticket is for the server to figure out if it is a rejoin automatically. 

Leave everything except check_and_start_local_server as is. In start_local_server, you can figure out if the database is up and do a rejoin if the db is up.

File: lib/python/vdm/server/voltdbserver.py [hunk 3]
```diff
@@ -109,7 +109,7 @@ def __init__(self, database_id):
         """
         self.database_id = database_id
 
-    def start_database(self, recover=False):
+    def start_database(self, recover=False, blocking=-1):
```
Comment (manjujames) [inline lib/python/vdm/server/voltdbserver.py]: This input parameter, blocking, in start_database is not used, correct? If so, remove it.

**Comment by manjujames** [comment]
The changes look good. Did you test it manually? Is there a way to add automated tests for this? If the tests will be unreliable, don't worry about it.

**Comment by deerwalk** [comment]
Yes, we have tested it manually. We will not be able to add automated tests for this task.

**Pull request merged**